user_profile/views.py

```diff
@@ -90,7 +93,14 @@ def edit_profile(request):
         )
         if profile_form.is_valid():
             profile_form.save()
```

Comment:
How does this show up? On the page or is it a logger with success message

Reply:
It shows up on the page. Django has a messages module, which, when called from a view function, queues up an alert style message that we can use to communicate with the user.  In base.html, we have some logic that will check if the messages data structure has anything in it. If it does, it'll iterate over the messages that are queued and display them at the top of the page.

<img width="693" alt="image" src="https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-5/assets/129902761/a1e361df-6153-4220-b655-996b6220912c">

Reply:
Nice that looks good